services/drupal/web/themes/epa_theme/source/_patterns/00-config/02-variables/_variables.external-urls.scss

```diff
@@ -1 +1 @@
-$external-urls: ":not([href=''], [href$='.gov'], [href*='.gov/'], [href$='.mil'], [href*='.mil/'], [href^='#'], [href^='?'], [href^='/'], [href^='.'], [href^='javascript:'], [href^='mailto:'], [href^='tel:'], [href*='webcms-uploads-dev.s3.amazonaws.com'], [href*='webcms-uploads-stage.s3.amazonaws.com'], [href*='webcms-uploads-prod.s3.amazonaws.com'], [href*='webcms-uploads-qa.s3.amazonaws.com'])";
+$external-urls: ":not([href=''], [href$='epa.gov'], [href*='epa.gov/'], [href^='#'], [href^='?'], [href^='/'], [href^='.'], [href^='javascript:'], [href^='mailto:'], [href^='tel:'], [href*='webcms-uploads-dev.s3.amazonaws.com'], [href*='webcms-uploads-stage.s3.amazonaws.com'], [href*='webcms-uploads-prod.s3.amazonaws.com'], [href*='webcms-uploads-qa.s3.amazonaws.com'])";
```

Comment:
btw, I know this will add the exit icon to all subdomains of epa.gov for CSS fallback, but that’s ok.

Reply:
Ok added the hard starts instead of the contains for the patterns. I left off the ^=epa.gov because those would render as internal links, so those would be broken to begin with and they'd still point to an internal page. Tested it with no JS running and it was correctly marking your additional test link as external.

Reply:
@jbwagner Hmm, `<a href="epa.gov">`  sends me to epa.gov in Firefox.

Reply:
And it looks like `<a href="//epa.gov">` works, so we should cover that use case too.

Reply:
Ah, interesting, Chrome didn't seem like it was going that for my testing. I've added epa.gov and //epa.gov for both bare and www URLs. Pushed up.